src/core/tests/visitors/op/max_pool.cpp

```diff
@@ -70,3 +70,38 @@ TEST(attributes, max_pool_v8_op) {
     EXPECT_EQ(g_max_pool->get_auto_pad(), max_pool->get_auto_pad());
     EXPECT_EQ(g_max_pool->get_index_element_type(), max_pool->get_index_element_type());
 }
+
+TEST(attributes, max_pool_v14_op) {
```

Comment:
Please add corresponding test for AvgPool-14.
https://github.com/openvinotoolkit/openvino/blob/27324189d5d5cb955e3837ce12bdb73c46a50404/src/core/tests/visitors/op/avg_pool.cpp#L70

Reply:
Added.